app/dashboard/templates/submit_bounty.html

```diff
@@ -53,9 +53,9 @@ <h5 class="font-subheader">{% trans "About" %}</h5>
                   <label class="form__label" for=issueURL>{% trans "Github Issue URL" %}</label>
                   <input name='issueURL' id="issueURL" class="form__input" type="url" placeholder="https://github.com/user/repo/issues/n" value="{% if issueURL %}{{issueURL}}{%endif%}" />
                   <label class="form__label" for="title">{% trans "Github Issue Title" %}</label>
-                  <input name='title' id="title"  class="form__input" type="text" placeholder="Deploy the Foo to The Bar" value="{{issue_title}}" readonly/>
+                  <input name='title' id="title"  class="form__input" type="text" placeholder="Deploy the Foo to The Bar" value="{{issue_title}}" title="Please update the title in the github issue" readonly/>
                   <label class="form__label" for="description">{% trans "Description" %}</label>
-                  <textarea name="description" class="form__input" id="description" placeholder="Add some new feature." readonly></textarea>
+                  <textarea name="description" class="form__input" id="description" placeholder="Add some new feature." title="Please update the description in the github issue" readonly></textarea>
```

Comment:
@SaptakS @thelostone-mc Can we have a followup PR to this for translation support?

Reply:
Got it boss :P

Reply:
Merged sensei